### PR TITLE
Move Pomm support section to data providers page

### PR DESCRIPTION
**Pull request by Pandoraaa**
| Q             | A
| ------------- | ---
| Bug fix?      | no 
| New feature?  | no 
| BC breaks?    | no
| Deprecations? | no
| Tickets       | fixes #934 
| License       | MIT
| Doc PR        | api-platform/docs#...

### CODE REVIEW

File: core/data-providers.md [hunk 1]
```diff
@@ -238,3 +238,8 @@ final class BlogPostItemDataProvider implements ItemDataProviderInterface, Restr
 }
 
 ```
+
+## Community data providers
```
Comment (alanpoulain) [inline core/data-providers.md]:
```suggestion
## Community Data Providers
```

Comment (alanpoulain) [reply core/data-providers.md]: Shouldn't it be at the top?

Comment (Pandoraaa) [reply core/data-providers.md]: :woman_shrugging:

Comment (dunglas) [reply core/data-providers.md]: At the bottom looks better to me!

File: core/data-providers.md [hunk 2]
```diff
@@ -238,3 +238,8 @@ final class BlogPostItemDataProvider implements ItemDataProviderInterface, Restr
 }
 
 ```
+
+## Community Data Providers
+
+If you don't want to use the built-in Doctrine system, alternative approaches which offer an integration with API Platform exist.
+For instance, [Pomm](http://www.pomm-project.org/) is a database access framework for PHP dedicated to PostgreSQL. The bundle to integrate it with API Platform can be found [here](https://github.com/pomm-project/pomm-api-platform).
```
Comment (dunglas) [inline core/data-providers.md]:
```suggestion
* [Pomm Data Provider](https://github.com/pomm-project/pomm-api-platform): ([Pomm](http://www.pomm-project.org/) is a database access framework dedicated to PostgreSQL database.
```

btw: https://www.w3.org/QA/Tips/noClickHere :)

Comment (Pandoraaa) [reply core/data-providers.md]: @dunglas Thanks for the link! ^^

ps: I'm blaming @alanpoulain on the click here :p

**Comment by alanpoulain** [comment]
After some thought, I'm not sure it's a good idea to have Pomm only here.
It's not just a Data Provider but also a set of filters.

**Comment by dunglas** [comment]
Technically, filters are a subpart of data providers, so it looks good to me.

**Comment by dunglas** [comment]
Thanks @Pandoraaa!

**Pull request merged**